lib/kamal/cli/proxy.rb

```diff
@@ -23,6 +23,7 @@ def boot
 
   desc "boot_config <set|get|reset>", "Manage kamal-proxy boot configuration"
   option :publish, type: :boolean, default: true, desc: "Publish the proxy ports on the host"
+  option :publish_ip, type: :string, desc: "IP address to bind HTTP/HTTPS traffic to. Defaults to all interfaces"
```

Comment:
Can we call this `publish_host_ip`?

Reply:
Absolutely, I think that is clearer - I'll fix that up.